pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/stats/SubscriptionStatsImpl.java

```diff
@@ -55,6 +55,9 @@
     /** Size of backlog in byte. **/
     public long backlogSize;
 
+    /** Get the time from the oldest message in the backlog to current time. */
+    public long timeBacklogInMills;
```

Comment:
From description, this timeBacklogInMills should be a time period as "current_timestamp"-"min_message_timestamp".
But it seems that it just set as min_message_timestamp?

And IMHO, name it as "backlogTime" would be more consistent with "backlogSize"

Reply:
+1

Reply:
fix